auth/usecase/usecase_test.go

```diff
@@ -29,12 +29,14 @@ func TestAuthFlow(t *testing.T) {
 		accountName = "user"
 		email       = "user@user.com"
 		suid, _     = uuid.NewRandom()
+		affiliateID = ""
```

Comment:
better some non-empty value, i.e `"affiliate_1"`

Reply:
fix in #70